aws_topology/tests/json/cloudtrail/sns_get_topic_attributes.json

```diff
@@ -0,0 +1,12 @@
+{
```

Comment:
I think I saw that GetTopicAttributes gives no more properties ListTopics (lets verify)

Reply:
It does give more attributes, fixed it